i18n/nl_NL.csv

```diff
@@ -166,7 +166,7 @@ No delay,Geen vertraging
 "This options allows you to set the number of days it takes you to pick, pack and hand in your parcels at PostNL when ordered before the cutoff time.","Stel hier het aantal dagen in die je nodig hebt om het product (te ontvangen,) te verpakken en aan te bieden bij PostNL, vanaf het moment dat de bestelling is geplaatst".
 The age check is intended for parcel shipments for which the recipient must show 18+ by means of a proof of identity. This option can't be combined with morning or evening delivery., De leeftijdscheck is bedoeld voor pakketzendingen waarbij de ontvanger moet kunnen aantonen dat hij/zij 18+ is. Deze optie kan niet worden gecombineerd met ochtend- of avondlevering.
 HS Codes are used for MyParcel world shipments, you can find the appropriate code on the site of the Dutch Customs,HS Codes worden gebruikt voor MyParcel wereldzendingen, je kunt de juiste code vinden op de site van de Douane.
-Fill in the amount of products that fit in a mailbox package. Set to 0 to automatically calculate based on weight. Regardless of this setting, the product will always be sent as a package if the weight is too high.,Vul het aantal producten in dat in een brievenbuspakket past. Stel in op 0 om automatisch te berekenen op basis van gewicht. Ongeacht deze instelling wordt het product altijd als pakket verzonden als het gewicht te hoog is.
+Fill in the amount of products that fit in a mailbox package. Use 0 to automatically calculate based on weight, -1 if the article does not fit in a mailbox. The product will always be sent as a package if the weight is too high.,Vul het aantal producten in dat in een brievenbuspakket past. Gebruik 0 om automatisch te berekenen op basis van gewicht, -1 wanneer het niet past. Het product wordt altijd als pakket verzonden als het gewicht te hoog is.
```

Comment:
Er zit een komma in de tekst, maar deze wordt ook gebruikt om de vertalingen te splitten... (hoe) werkt dit? 😬 Er moeten quotes omheen lijkt me, maar dat was eerst ook al niet zo

Overigens zou ik ook iets concreter zijn in de tekst:

```suggestion
"Fill in the amount of products that fit in one mailbox package. Use 0 to automatically calculate based on weight, -1 if the article does not fit in a mailbox package. The product will always be sent as a regular package if it's too heavy for a mailbox package.","Vul het aantal producten in dat in één brievenbuspakket past. Gebruik 0 om automatisch te berekenen op basis van gewicht, -1 wanneer het niet in een brievenbuspakket past. Het product wordt altijd als pakket verzonden als het gewicht te hoog is voor een brievenbuspakket.
"
```